test/e2e_source_map_test.ts

```diff
@@ -310,6 +310,29 @@ function createTests(useTransformer: boolean) {
           .to.equal('input2.ts', 'input file name');
     }
   });
+
+  it('maps at the start of lines correctly', () => {
+    const sources = new Map<string, string>();
```

Comment:
FYI you can do: `new Map([['input.ts', 'code goes here']])` (even with type inference!)